sklearn/model_selection/tests/test_search.py

```diff
@@ -245,76 +246,35 @@ def test_grid_search_labels():
         gs.fit(X, y)
 
 
-def test_trivial_grid_scores():
-    # Test search over a "grid" with only one point.
-    # Non-regression test: grid_scores_ wouldn't be set by GridSearchCV.
+def test_trivial_results_attr():
+    # Test search over a "grid" with only one point. (NRT for gh-1782)
     clf = MockClassifier()
     grid_search = GridSearchCV(clf, {'foo_param': [1]})
     grid_search.fit(X, y)
-    assert_true(hasattr(grid_search, "grid_scores_"))
+    assert_true(hasattr(grid_search, "results_"))
 
     random_search = RandomizedSearchCV(clf, {'foo_param': [0]}, n_iter=1)
     random_search.fit(X, y)
-    assert_true(hasattr(random_search, "grid_scores_"))
+    assert_true(hasattr(grid_search, "results_"))
 
 
 def test_no_refit():
-    # Test that grid search can be used for model selection only
+    # Test that GSCV can be used for model selection alone without refitting
     clf = MockClassifier()
     grid_search = GridSearchCV(clf, {'foo_param': [1, 2, 3]}, refit=False)
     grid_search.fit(X, y)
-    assert_true(hasattr(grid_search, "best_params_"))
+    assert_true(hasattr(grid_search, "best_index_"))
```

Comment:
Should we also check:

    assert_false(hasattr(grid_search, "best_estimator_"))